twine-cert.txt

```diff
@@ -1616,8 +1616,8 @@
     en = Full recovery
 
 [certificates_overview_recovery_certificate_valid_until_date]
-    de = Höchstens gültig bis %@
-    en = Maximum valid until %@
+    de = Positiver Test vom %@
```

Comment:
@RaulMeer  we should use a new key for this, in order to be backward compatible and have matching key-value pairs